### PR TITLE
[BUG]Fix read operations to support tags 

### LINKED ISSUE #24: [BUG]Fix read operations to support tags

### PR DESCRIPTION
**Pull request by RupeshPatil20**
Closes #24

### CODE REVIEW

File: helpers/tags.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+func TagsSchemaComputed() *schema.Schema {
+	return &schema.Schema{
+		Type:     schema.TypeMap,
+		Optional: true,
+		Computed: true,
+	}
+}
+
+func TagsSchemaForceNew() *schema.Schema {
```
Comment (kaidaguerre) [inline helpers/tags.go]: needed?

File: helpers/tags.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+// tagsSchema returns the schema to use for tags.
+//
+func TagsSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:     schema.TypeMap,
+		Optional: true,
+	}
+}
+
+func TagsSchemaComputed() *schema.Schema {
```
Comment (kaidaguerre) [inline helpers/tags.go]: needed?

File: helpers/tags.go (hunk 1 above)
Comment (RupeshPatil20) [reply helpers/tags.go]: I have removed it

File: helpers/tags.go (hunk 2 above)
Comment (RupeshPatil20) [reply helpers/tags.go]: Removed

**Pull request merged**